lib/guessing_game.js

```diff
@@ -0,0 +1,28 @@
+var pry = require('pryjs');
+function GuessingGame(num){
+  var num = num;
+  var solvedStatus = false
+
+  this.isSolved = function(swap){
+    
+    if(swap === "got it"){
+      solvedStatus = true
+    }
+    return solvedStatus
+  }
+
+  this.guess = function(userGuess) {
```

Comment:
yes! @CjMoore that's exactly how it would work since we've created a constructor function with `GuessingGame` - adding functions to the prototype is also preferred. see if you can add it to the prototype and run your tests to make sure they pass 👍